### PR TITLE
it-IT: Updated strings

### PR DESCRIPTION
**Pull request by LuigiPiccoli17**
Updated some strings in the Italian translation

### CODE REVIEW

File: data/language/it-IT.txt [hunk 1]
```diff
@@ -1879,8 +1879,8 @@ STR_1872    :{COMMA16}
 STR_1873    :{WINDOW_COLOUR_2}Entrate: {BLACK}{CURRENCY} all'ora
 STR_1874    :{WINDOW_COLOUR_2}Profitto: {BLACK}{CURRENCY} all'ora
 STR_1875    :{BLACK} {SPRITE}{BLACK} {STRINGID}
-STR_1876    :{WINDOW_COLOUR_2}{INLINE_SPRITE}{251}{19}{00}{00}Ispeziona attrazioni
-STR_1877    :{WINDOW_COLOUR_2}{INLINE_SPRITE}{252}{19}{00}{00}Aggiusta attrazioni
+STR_1876    :{WINDOW_COLOUR_2}{INLINE_SPRITE}{251}{19}{00}{00}Ispeziona le attrazioni
```
Comment (Gymnasiast) [inline data/language/it-IT.txt]: These strings are used to select what a mechanic should do:

scr202

Can you check if your translation is correct in this context?

**Comment by LuigiPiccoli17** [comment]
Hello Gymnasiast,
I am really new to GitHub and I am trying to learn how it works :) I have in fact checked in the italian RCT that the original translation is the most recent one

**Comment by Gymnasiast** [comment]
Sure, but the original translation is not always correct. So it's not that important whether a translation is original or not.

**Comment by LuigiPiccoli17** [comment]
Ok, thank you!

**Pull request merged**